adr/0006-supported-node-and-npm-versions.md

```diff
@@ -0,0 +1,43 @@
+# Supported Node Versions
+
+Date: 2017-12-15
+
+## Status
+
+Proposed
+
+## Context 
+
+eLife has projects built in Node.
+ 
+eLife has projects that use the Node Package Manager (npm) in at least part of their own build process, even if the main technology of the project is not Node.
+
+In order to provide language version consistency across projects we need to get a consensus on which Node versions we are going to support.
+
+In order to provide dependency management consistency, we need a consensus on which npm versions to support.
+
+Staying up to date with the major and minor versions of Node is important:
+
+- to be able to use new features (nice to have)
+- to keep working on a version that receives bug fixes and security updates (must have)            
+
+## Decision
+
+To use the latest Long Term Support (LTS) version.
+
+The 8.x is the current Node LTS line. The current Node LTS version is 8.9.3.
```

Comment:
I agree on including `8.x` as the actual `LTS` changes as time passes, and we should revise the decision after that has changed. I think minor and patch versions are too detailed for an ADR instead, their place is in formulas and similar automation configuration.